src/plugins/files/server/saved_objects/file.ts

```diff
@@ -51,7 +51,7 @@ const properties: Properties = {
 export const fileObjectType: SavedObjectsType<FileMetadata> = {
   name: FILE_SO_TYPE,
   hidden: true,
-  namespaceType: 'agnostic',
+  namespaceType: 'multiple-isolated',
```

Comment:
Reading the docs:

> In other words, objects with a "multiple-isolated" namespace type will be *share-capable*

I guess that means that are not shareable right now? CC @rudolf

Reply:
Correct. The security team has docs on what needs to be done to make a `multiple-isolated` saved object shareable.